CODIGO/Protocol.bas

```diff
@@ -5753,33 +5735,26 @@ Private Sub HandleMeditateToggle()
     
     If charindex = UserCharIndex Then
         UserMeditar = (fX <> 0)
-        
         If UserMeditar Then
-
             With FontTypes(FontTypeNames.FONTTYPE_INFO)
                 Call ShowConsoleMsg("Comienzas a meditar.", .red, .green, .blue, .bold, .italic)
-
             End With
-
         Else
-
             With FontTypes(FontTypeNames.FONTTYPE_INFO)
                 Call ShowConsoleMsg("Has dejado de meditar.", .red, .green, .blue, .bold, .italic)
-
             End With
-
         End If
-
     End If
     
     With charlist(charindex)
-
         If fX <> 0 Then
             Call StartFx(.ActiveAnimation, Fx, -1)
+            Call ao20audio.PlayWav(158, True, ao20audio.ComputeCharFxVolume(.Pos), ao20audio.ComputeCharFxPan(.Pos), "meditate" & CStr(CharIndex))
```

Comment:
can we define a constant to sound "158"?

Reply:
done in the latest patch